docker-compose.yml

```diff
@@ -8,8 +8,12 @@ services:
     build: ./kafka
     ports:
       - "9092:9092"
+      - "29092:29092"
```

Comment:
it's needed, because we use two hostnames (localhost for clients external to the Docker network, kafka for client within to the Docker network). I can try to make one hostname, if necessary.